src/skate.js

```diff
@@ -19,7 +19,19 @@ import version from './version';
  * @returns {undefined}
  */
 var initDocument = debounce(function () {
-  initElements(document.getElementsByTagName('html'));
+  if (hasDomContentLoadedFired) {
+    initElements(document.getElementsByTagName('html'));
```

Comment:
Should probably consider using `document.documentElement` instead of using `getElementsByTagName`. I know this existed before, but if you could make that change as well it'd be nice.